docs/release-notes/index.rst

```diff
@@ -0,0 +1,287 @@
+.. _release-notes:
+
+Release notes
+=============
+
+This page lists the notes for past releases of Snapcraft, which summarise new features,
+bug fixes and backwards-incompatible changes in each version. It also contains the
+release and support policies for Snapcraft.
+
+
+Current releases
+----------------
+
+Snapcraft 8
+~~~~~~~~~~~
+
+- :ref:`Snapcraft 8.7 <release-8.7>`
+
+
+Snapcraft 7
+~~~~~~~~~~~
+
+Snapcraft 7 is available for building ``core18`` snaps. When building for newer bases,
+use Snapcraft 8. See
+:ref:`How to install multiple instances of Snapcraft <multiple-installs>` for
+more information.
+
+- :ref:`Snapcraft 7.5.8 <changelog-7-5-8>`
```

Comment:
This is a puzzle. I imagine we'd need an equivalent of 7.5 in the changelog for this to be maintainable and consistent, otherwise you'll be bumping this with every release. I suppose when we release 7.5.9 or 7.6, you'll write release notes for it?

Reply:
I agree - I will add release notes with the next 7 release (7.5.9 or 7.6).